spdx-utils/src/main/kotlin/SpdxExpression.kt

```diff
@@ -491,6 +533,20 @@ data class SpdxLicenseReferenceExpression(
     override fun getLicenseUrl(): String? = null
 }
 
+/**
+ * Aggregates multiple SpdxExpressions to a [SpdxCompoundExpression] concatenated using [SpdxOperator.OR].
+ */
+fun List<SpdxExpression>.choicesToSpdx(): SpdxExpression {
+    if (size == 1) return this[0]
+
+    var result: SpdxExpression = this[0]
+    this.subList(1, this.size).forEach {
```

Comment:
We prefer to drop `this.` if not strictly required.

Reply:
Not relevant anymore